Neos.Neos/Resources/Private/Fusion/Prototypes/DimensionsMenu.fusion

```diff
@@ -1,25 +1,32 @@
 # Neos.Neos:DimensionsMenu provides dimension (e.g. language) menu rendering
 prototype(Neos.Neos:DimensionsMenu) < prototype(Neos.Neos:Menu) {
-	@class = 'Neos\\Neos\\Fusion\\DimensionsMenuImplementation'
-
 	templatePath = 'resource://Neos.Neos/Private/Templates/FusionObjects/DimensionsMenu.html'
 
 	# the "absent" state is assigned to items for dimension (combinations) for which no node variant exists
 	absent.attributes = Neos.Fusion:Attributes {
 		class = 'normal'
 	}
 
-	# if documents which are hidden in index should be rendered or not
-	renderHiddenInIndex = true
-
 	# name of the dimension to use (optional)
-	# dimension = 'language'
+	dimension = null
 
 	# list of presets, if the default order should be overridden, only used with "dimension" set
-	# presets = ${['en_US']}
+	presets = null
```

Comment:
Should the default maybe an empty array so that we don't have to type check?
```suggestion
	presets = ${[]}
```
not sure though

Reply:
Actually it would decide later to skip if this was an empty array instead of null. Since the logic is not touched here i would not adjust this in this pr.